Add DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED's doc (#17718)
DataDog/documentation: content/en/tracing/trace_collection/library_config/php.md

**Default**: `*`<br>
A comma-separated list of query parameters to be collected as part of the URL. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.

`DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED`
: **INI**: `datadog.trace.http_post_data_param_allowed`<br>
**Default**: ""<br>

> A comma-separated list of HTTP POST data fields to be collected. Leave empty if you don't want to collect any posted values. When setting this value to the wildcard `*`, all posted data will be collected, but the values for fields that match the `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` obfuscation rule will be redacted. If specific fields are given, then only these fields' values are collected in clear, while the values for all other fields will be redacted. Added in version `0.86.0`.<br>

Reviewer: ```suggestion
A comma-separated list of HTTP POST data fields to be collected. Leave empty if you don't want to collect any posted values. When setting this value to the wildcard `*`, all posted data is collected, but the values for fields that match the `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` obfuscation rule are redacted. If specific fields are given, then only these fields' values are visible, while the values for all other fields are redacted. Added in version `0.86.0`.<br>
```

**Example**: 
  - The posted data is `foo[baz]=bar&foo[baz][bar]=quz&foo[bar][password]=baz&foo[bar][baz]=quz`
  - `DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED` is set to `foo.baz,foo.bar.password`<br>

>   - In this scenario, the collected metadata is:

Reviewer 2: Are you sure of the example @PROFeNoM ? I would have assumed something different

Author: Yes, there was indeed something wrong

**Default**: `*`<br>
A comma-separated list of query parameters to be collected as part of the URL. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.

`DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED`
: **INI**: `datadog.trace.http_post_data_param_allowed`<br>
**Default**: ""<br>

> A comma-separated list of HTTP POST data fields to be collected as part the request. Set to empty by default to redact all posted values, or `*` to collect and redact them if needed. When specific fields are explicitly whitelisted, only those fields' values aren't redacted. Added in version `0.86.0`.<br>

Reviewer: ```suggestion
A comma-separated list of HTTP POST data fields to be collected as part the request. Leave empty to redact all posted values, or use `*` to collect and redact them if needed. When specific fields are explicitly allowed, only those fields' values remain. Added in version `0.86.0`.<br>
```

**Example**: 
  - The posted data is `foo[baz]=bar&foo[baz][bar]=quz&foo[bar][password]=baz&foo[bar][baz]=quz`
  - `DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED` is set to `foo.baz,foo.bar.password`<br>
  - In this scenario, the collected metadata is:
    - `http.request.foo.baz=bar`
    - `http.request.foo.baz.bar=quz`
    - `http.request.foo.bar.password=baz`
    - `http.request.foo.bar.baz=<redacted>`

> 

Reviewer: ```suggestion
```

`DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED`
: **INI**: `datadog.trace.http_post_data_param_allowed`<br>
**Default**: ""<br>

> A comma-separated list of HTTP POST data fields to be collected as part the request. Leave empty to redact all posted values, or use `*` to collect and redact them if needed. When specific fields are explicitly allowed, only those fields' values remain. Added in version `0.86.0`.<br>

Reviewer: ```suggestion
A comma-separated list of HTTP POST data fields to be collected as part the request. Leave empty to redact all posted values, or use `*` to collect them all and redact if needed. When specific fields are explicitly allowed, only those fields' values remain. Added in version `0.86.0`.<br>
```
